tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -54,6 +54,18 @@ internal static class Debugger
             /// <seealso cref="DebuggerSettings.SymbolDatabaseUploadEnabled"/>
             public const string SymbolDatabaseUploadEnabled = "DD_DYNAMIC_INSTRUMENTATION_SYMBOL_DATABASE_UPLOAD_ENABLED";
 
+            /// <summary>
+            /// Configuration key for a separated comma list of libraries to include in the 3rd party detection
+            /// Default value is empty.
+            /// </summary>
+            public const string ThirdPartyDetectionIncludes = "DD_THIRD_PARTY_DETECTION_INCLUDES";
```

Comment:
How is this different from `DD_SYMBOL_DATABASE_THIRD_PARTY_DETECTION_INCLUDES`?

Reply:
`DD_THIRD_PARTY_DETECTION_INCLUDES` doesn't seem to follow the usual naming conventions of `DD_<PRODUCT>_NAME`... can we improve that?

Reply:
> How is this different from DD_SYMBOL_DATABASE_THIRD_PARTY_DETECTION_INCLUDES?
The one with `SYMBOL_DATABASE` overrides the default one.

> DD_THIRD_PARTY_DETECTION_INCLUDES doesn't seem to follow the usual naming conventions of DD_<PRODUCT>_NAME... can we improve that?
The thing is that `THIRD_PARTY_DETECTION` is not specific for a product, i.e. it is not a Dynamic Instrumentation only.

Reply:
It's related to "symbol database" which is a "product", no? 🤔

Reply:
Not exactly, it's also useable for exception reply for example (and maybe for more products)

Reply:
Not exactly, third party detection can be useful for skipping frames for example (so exception reply can use it) or to decide if we it's allow to put probe in third party code.

Reply:
I dunno, I still feel like this is a _really_ confusing set of configuration values for customers 🤔 Have we agreed all these across languages? Personally I would have no idea how to use them based on the configuration names... Are we sure we're happy with all these @OmerRaviv?  I'll assume so and approve though